EvernoteOAuth.lua

```diff
@@ -200,11 +200,15 @@ function OAuth:loadPage(method, path, params, data)
   request['source'] = method == "POST" and ltn12.source.string(query) or nil
   request['sink'] = ltn12.sink.table(sink)
   request['headers'] = headers
+  logger.dbg(request)
 
   http.TIMEOUT, https.TIMEOUT = 10, 10
   local httpRequest = parsed.scheme == 'http' and http.request or https.request
   local code, headers, status = socket.skip(1, httpRequest(request))
 
+  logger.dbg("Evernote received response: ")
+  logger.dbg("Response code: ", code)
+  logger.dbg("Response headers: ", headers)
```

Comment:
Not exactly required but you can merge these statements into a single one:

```lua
    logger.dbg("Evernote received response:", "Response code:", code, "Response headers", headers)
```

Reply:
Won't new lines be missing this way? I think it's better to keep this as is, while it's possible to add '\n' after every line, I think it's less readable.